server.js

```diff
@@ -1,30 +1,96 @@
-import express from "express";
-import cors from "cors";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
+import express from "express"
+import cors from "cors"
+import expressListEndpoints from "express-list-endpoints"
+
+//Import data from sneaker json file
+import sneakerData from "./data/sneakers.json"
+
+// Defines the port the app will run on
+const port = process.env.PORT || 8080
+const app = express()
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+app.use(cors())
+app.use(express.json())
 
-// Start defining your routes here
+// Route handler
+//http://localhost:8080/
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+  const endpoints = expressListEndpoints(app)
+  res.json(endpoints)
+})
```

Comment:
would be good to add to the documentation how to filter data using the queries like:

```javascript
app.get("/", (req, res) => {
  const endpoints = expressListEndpoints(app);
  const updatedEndpoints = endpoints.map((endpoint) => {
    if (endpoint.path === '/sneakers') {
      endpoint.query = ['name', 'brand', 'color', 'price']; // Add the query parameters for sneakers endpoint
    }
    return endpoint;
  });
  res.json(updatedEndpoints);
});
``` 

This way we inform other devs about how to fully use the api! ⭐